Tests/UnitTests/Networking/HTTPClientTests.swift

```diff
@@ -289,13 +285,13 @@ class HTTPClientTests: XCTestCase {
                                      headers: nil)
         }
 
-        self.client.perform(request, authHeaders: [:]) { (status, result) in
-            isResponseCorrect.value = (status == .success) && (result.error == nil)
-            message.value = result.value?["message"] as? String
+        self.client.perform(request, authHeaders: [:]) {
+            result.value = $0
         }
 
-        expect(message.value).toEventually(equal("something is great up in the cloud"), timeout: .seconds(1))
-        expect(isResponseCorrect.value) == true
+        expect(result.value).toEventuallyNot(beNil(), timeout: .seconds(1))
+        expect(result.value).to(beSuccess())
+        expect(result.value?.value?.jsonObject["message"] as? String) == "something is great up in the cloud"
```

Comment:
A future PR will make `HTTPClient` not return `.success` when requests fail. This doesn't change the test, but soon this will become `expect(result.value).to(beFailure())` instead.

Reply:
#1431